doc/index.rst

```diff
@@ -63,14 +63,13 @@ Project and community
 =====================
 
 Subiquity is a member of the Ubuntu family. It's an open source project that
-warmly welcomes community projects, contributions, suggestions, fixes, and
+warmly welcomes community projects, contributions, suggestions, fixes and
 constructive feedback.
 
-* Read our `Code of Conduct`_
-* IRC?
-* Discourse?
-* Contribute?
-* Roadmap?
+* Read our `Code of Conduct`_.
+* IRC: `Libera.Chat`_ (the Foundations team meets every week on Thursdays (16:00 UTC) in the *#ubuntu-meeting* channel).
```

Comment:
We could point to `#ubuntu-server` here, but mentioning the meeting is not what we want here.  This is a meeting for the foundations team that is held in public for visibility - note the text listed on the calendar "The Ubuntu Foundations Team meets regularly to talk about status updates, release incoming bugs, merges and other important work. Drop by #ubuntu-meeting to **see** what they are up to."  Other venues are intended for bugs or discussion.

Reply:
Thank you. Changed.